.github/workflows/build_homebrew.yml

```diff
@@ -0,0 +1,36 @@
+name: Build Homebrew formula
+
+on:
+  release:
+    types:
+    - created
+
+jobs:
+  build:
+    name: Build Homebrew formula
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Build formula
+        run: |
+          utility/build-homebrew.sh cumulusci.rb
+      - name: Test formula
+        run: |
+          brew audit cumulusci.rb
+          brew install cumulusci.rb
+          brew test cumulusci.rb
+      - name: Create branch in homebrew-sfdo repository
```

Comment:
Neat. If this works we'll toss `utility/push-homebrew.py`?

Reply:
@jstvz I think I'll need to iterate on this, but yeah.